frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText.unit.spec.tsx

```diff
@@ -105,5 +85,10 @@ function setup(additionalProps?: Partial<ExpressionEditorHelpTextProps>) {
 
   render(<ExpressionEditorHelpText {...props} />);
 
+  // have to wait for TippyPopover to render content
+  await waitFor(() => {
+    expect(screen.getByTestId("expression-helper-popover")).toBeInTheDocument();
```

Comment:
I'm not sure is it a case here, but my understanding is to use `waitFor` when I need to be sure function is called (e.g. toHaveBeenCalled), but `expect(await screen.findbBy...).toBeInTheDocument()` when we work with DOM

do you know why  `await waitFor` can be better than `findBy`?